lib/galaxy/config/__init__.py

```diff
@@ -636,7 +636,7 @@ def _process_config(self, kwargs):
         # InteractiveTools propagator mapping file
         self.interactivetools_map = self.resolve_path(kwargs.get("interactivetools_map", os.path.join(self.data_dir, "interactivetools_map.sqlite")))
         self.interactivetool_prefix = kwargs.get("interactivetools_prefix", "interactivetool")
-        self.interactivetool_proxy_host = kwargs.get("interactivetool_proxy_host", None)
+        self.interactivetool_proxy_host = kwargs.get("interactivetools_proxy_host", None)
```

Comment:
Should the attribute be renamed too? (here and wherever is used)

Reply:
This matches the pattern of prefix - but it looks like prefix is just used in one place so I can change both of them.

Reply:
Yep, I think that would help @ic4f's config refactorings too.

Reply:
Yes, it would! If we can keep those identifies in sync with the schema, those 3 lines can be removed (separate PR - after integration testing for those and other config properties works correctly) Thank you!